tests/test_imageDecorrelation.py

```diff
@@ -18,11 +18,8 @@
 # You should have received a copy of the LSST License Statement and
 # the GNU General Public License along with this program.  If not,
 # see <https://www.lsstcorp.org/LegalNotices/>.
-from __future__ import absolute_import, division, print_function
 import unittest
 
-from builtins import range
-from past.builtins import basestring
```

Comment:
As you are doing more than future, you may consider changing the commit message to something about dropping py2 support and syntax, but it is just a suggestion